client/ayon_core/hosts/houdini/plugins/load/load_alembic.py

```diff
@@ -59,7 +59,7 @@ def load(self, context, name=None, namespace=None, data=None):
 
         normal_node.setInput(0, unpack)
 
-        null = container.createNode("null", node_name="OUT".format(name))
+        null = container.createNode("null", node_name="OUT")
```

Comment:
As a formatting change it is okay, but I am wondering if it shouldn't be adding name into the node name to make it more unique, but I don't know anything about houdini integration.

Reply:
This is actually opted for removal in #253 - so I'd say disregard.
Also, it's always "unique" inside the container itself, so no need to add extra names, etc.

This change here is thus OK as is.